src/redux/actions/profile.action.js

```diff
@@ -0,0 +1,17 @@
+import { updateUserProfile } from "../../api/profileApi";
+import { updatedProfile } from "../features/profile.feature";
+import { registerSuccess, registerFail } from '../features/auth.feature';
```

Comment:
Remove this register features as you are not using them

Reply:
I am using them to register status and messages